File: clients/swift/src/Models/Space.swift
import Foundation

public struct SpaceObject: Codable, Sendable {
    public let id: Uid
}

public struct Space: Codable, Sendable {
    public let id: Uid
    public let name: String
    public let color: Color
    public let created: Date
    public let objects: [SpaceObject]
}

File: clients/typescript/src/models/Space.ts
export interface SpaceObject {
  id: string;
}

export interface Space {
  id: string;
  /** Display name. Unique across the user's spaces. */
  name: string;
  /** Display color — any valid CSS color value, typically a hex code. */
  color: string;
  /** ISO 8601 UTC timestamp of when the space was created. */
  created: string;
  objects: SpaceObject[];
}
